web/src/components/Feedback.tsx

```diff
@@ -56,8 +56,8 @@ const Feedback = ({
   const isSearchFeedback = searchTerm !== undefined
   const commentTitle = isSearchFeedback ? 'wantedInformation' : 'commentHeadline'
   const [privacyPolicyAccepted, setPrivacyPolicyAccepted] = useState(false)
-  const feedbackFilled = rating === null && comment.trim().length === 0 && !searchTerm
-  const submitFeedbackDisabled = feedbackFilled || !privacyPolicyAccepted
+  const feedbackMissing = rating === null && comment.trim().length === 0 && !searchTerm
```

Comment:
🙃  `isFeedbackMissing`

Reply:
Personally I like it better without :) Since we don't have a guideline for this, I'll keep it as it is.